manticore/platforms/linux.py

```diff
@@ -793,21 +809,21 @@ def update(address, code):
 
         self.current.memory.mmap(0xFFFF0000, len(page_data), "r x", page_data)
 
-    def load_vdso(self, bits):
-        # load vdso #TODO or #IGNORE
-        vdso_top = {32: 0x7FFF0000, 64: 0x7FFF00007FFF0000}[bits]
-        with open(f"vdso{bits:2d}.dump") as f:
-            vdso_size = len(f.read())
-        vdso_addr = self.memory.mmapFile(
-            self.memory._floor(vdso_top - vdso_size),
-            vdso_size,
-            "r x",
-            {32: "vdso32.dump", 64: "vdso64.dump"}[bits],
-            0,
-        )
-        return vdso_addr
-
-    def setup_stack(self, argv, envp):
+    # def load_vdso(self, bits: int):
+    #     # load vdso #TODO or #IGNORE
+    #     vdso_top = {32: 0x7FFF0000, 64: 0x7FFF00007FFF0000}[bits]
+    #     with open(f"vdso{bits:2d}.dump") as f:
+    #         vdso_size = len(f.read())
+    #     vdso_addr = self.current.memory.mmapFile(
+    #         self.current.memory._floor(vdso_top - vdso_size),
+    #         vdso_size,
+    #         "r x",
+    #         {32: "vdso32.dump", 64: "vdso64.dump"}[bits],
+    #         0,
+    #     )
+    #     return vdso_addr
```

Comment:
Note: `load_vdso` was dead code previously, and had type error in it also.

Reply:
From slack by @feliam 

> AUXV were selected to avoid vdso

If it's dead code, we should probably just delete 🤷‍♂